test/performance/odp_sched_perf.c

```diff
@@ -167,6 +181,12 @@ static void print_usage(void)
 	       "                         if num_cpu is multiple of num_group and num_group is multiple of num_join.\n"
 	       "                         0: join all groups (default)\n"
 	       "  -b, --burst            Maximum number of events per operation. Default: 100.\n"
+	       "  -C, --cache_stash      Enable cache stashing. Format: offset,len,region,level\n"
```

Comment:
"Enable cache stashing" could be "Enable common group level cache stashing" to make it clear that this configures common stuff and does not touch priority specific stashing.

Also, matter of taste, but the format "region,level,offset,len" might be more natural for users.